tree/tree/test/BulkApiSillyStruct.cxx

```diff
@@ -25,34 +25,31 @@ class BulkApiSillyStructTest : public ::testing::Test {
 protected:
    void SetUp() override
    {
-      TFile *hfile = new TFile(fFileName.c_str(), "RECREATE", "TTree silly-struct benchmark");
-      hfile->SetCompressionLevel(0); // No compression at all.
+      TFile hfile{fFileName.c_str(), "RECREATE", "TTree silly-struct benchmark"};
+      hfile.SetCompressionLevel(0); // No compression at all.
 
-      TTree *tree = new TTree("T", "A ROOT tree of silly-struct branches.");
-      tree->SetBit(TTree::kOnlyFlushAtCluster);
-      tree->SetAutoFlush(fClusterSize);
+      TTree tree{"T", "A ROOT tree of silly-struct branches."};
+      tree.SetBit(TTree::kOnlyFlushAtCluster);
+      tree.SetAutoFlush(fClusterSize);
       ROOT::TIOFeatures features;
       features.Set(ROOT::Experimental::EIOFeatures::kGenerateOffsetMap);
-      tree->SetIOFeatures(features);
+      tree.SetIOFeatures(features);
 
       SillyStruct ss;
-      TBranch *branch = tree->Branch("myEvent", &ss, 32000, 99);
+      TBranch *branch = tree.Branch("myEvent", &ss, 32000, 99);
       branch->SetAutoDelete(false);
 
       Int_t nb = 0;
       for (Long64_t ev = 0; ev < fEventCount; ev++) {
          ss.i = ev;
          ss.f = ev;
          ss.d = ev;
-         nb += tree->Fill();
-      } 
+         nb += tree.Fill();
+      }
 
-      hfile = tree->GetCurrentFile();
-      hfile->Write();
-      tree->Print();
+      hfile.Write();
```

Comment:
Note: the pattern:
```
hfile = tree->GetCurrentFile();
```
is necessary in case the mechanism `TTree::ChangeFile` is automatically triggered (file larger than the requested limit).  If this is not longer supported, this is a problem.

Reply:
Ok will change this